eslint.config.mjs

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license Copyright (c) 2003-2025, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import globals from 'globals';
+import { defineConfig } from 'eslint/config';
+import ckeditor5Config from 'eslint-config-ckeditor5';
+import ckeditor5Rules from 'eslint-plugin-ckeditor5-rules';
```

Comment:
Shouldn't both packages be included in the root `package.json` file as `devDependencies`?

Reply:
`globals` too, I guess.

Reply:
Done in https://github.com/ckeditor/ckeditor5-linters-config/pull/60/commits/ebf1445d16be3f85d17b33a9b52c064bbea5c757.